### PR TITLE
Chinese translation file for 0.7

### PR DESCRIPTION
**Pull request by silentdark**
Chinese translation file for 0.7

### CODE REVIEW

File: resources/l10n/zh_CN/LC_MESSAGES/loot.po [hunk 1]
```diff
@@ -1,2167 +1,921 @@
...
-#: backend/game.cpp:248 backend/game.cpp:308 backend/game.cpp:363
+#: src/backend/game.cpp:489
+msgid ""
+"libloadorder failed to create a game handle. Details could not be fetched."
+msgstr "加载顺序数据库无法创建游戏处理器，详细信息无法获取。"
+
+#: src/backend/game.cpp:493
 msgid "libloadorder failed to create a game handle. Details:"
-msgstr "libloadorder 无法创建游戏处理器，详细信息："
+msgstr "加载顺序数据库无法创建游戏处理器，详细信息："
```
Comment (WrinklyNinja) [inline resources/l10n/zh_CN/LC_MESSAGES/loot.po]: `libloadorder` is a "proper noun", it is the name of a specific software library, so it shouldn't be translated.

File: resources/l10n/zh_CN/LC_MESSAGES/loot.po [hunk 2]
```diff
@@ -1,2167 +1,921 @@
...
+#: src/backend/metadata.cpp:886
+msgid "This plugin requires \"%1%\" to be active, but it is inactive."
+msgstr "这个插件激活\"%1%\"。"
 
-#: gui/settings.cpp:298
-msgid "Error: A game with this name is already defined. Row will not be added."
-msgstr "错误：已存在此名字的游戏，列不会被添加。"
+#: src/backend/metadata.cpp:901
+msgid "This plugin is incompatible with \"%1%\", but both are present."
+msgstr "这个插件\"%1%\"与\"%2%\"不兼容，但是两者都存在。"
```
Comment (WrinklyNinja) [inline resources/l10n/zh_CN/LC_MESSAGES/loot.po]: There is no `%2%` in the original text.

**Comment by WrinklyNinja** [comment]
Thanks for your contribution! I'll merge it in once the issues I've commented on above have been resolved.

**Comment by silentdark** [comment]
I have fixed those issues, thank you.

**Pull request merged**